lib/pdf_fill/forms/va210781v2.rb

```diff
@@ -790,10 +822,12 @@ def format_event(event, index)
         event_overflow.push("Event Location: \n\n#{event_location}")
         event_overflow.push("Event Date: \n\n#{event_timing}")
 
+        %w[details location timing].each { |key| event[key] = nil } if overflow_only
```

Comment:
It's a little unfortunate, but turning on the structured data in the class constant `KEY` (see above annotation around L150) means the old-style concatenator has to suppress the structured data in code here. All so that there's no change in how the old overflow page renders.